src/components/tabBars.js

```diff
@@ -78,26 +78,30 @@ export const TabBar = ({
   }
 
   return div({
-    role: 'navigation',
-    'aria-label': props['aria-label'], // duplicate the menu's label on the navigation element
-    'aria-labelledby': props['aria-labelledby']
+    style: Style.tabBar.container
   }, [
-    h(HorizontalNavigation, {
-      role: 'menu',
-      'aria-orientation': 'horizontal',
-      style: Style.tabBar.container,
-      ...props
+    div({
+      role: 'navigation',
```

Comment:
Only want the navigation role around the menu bar (don't want to include children on the right-hand side within the navigation role).